CP-CPS.md

```diff
@@ -228,9 +228,7 @@ No stipulation.
 
 ### 3.2.2 Authentication of organization identity
 
-ISRG only issues Domain Validation (DV) certificates. All FQDNs which will be listed in the Common Name and list of SANs in the certificate are fully validated prior to issuance.
-
-ISRG uses three methods for validating domain control:
+Prior to issuance of a Subscriber Certificate, ISRG uses at least one of the following methods to validate the Applicant's control of each FQDN listed in the Certificate:
```

Comment:
I should have caught this before, but as in another PR - strictly speaking, this is saying we perform validation on *every* FQDN listed in the certificate. That isn't technically true though because of things like OCSP.

The fix for my last comment looks good though.

Reply:
Yes, but it's also reflecting the language [used by the BRs](https://github.com/cabforum/servercert/blob/main/docs/BR.md#3224-validation-of-domain-authorization-or-control):

> The CA SHALL confirm that prior to issuance, the CA has validated each Fully-Qualified Domain Name (FQDN) listed in the Certificate...

Reply:
I'm OK with mimicking that if that seems like the best thing to do, but it makes me uncomfortable that the plain English reading of the BRs here seems incorrect.

Reply:
I asked around a bit and people seemed to think contextual interpretation is sufficient, but after some further thought I don't think there is a compelling reason for us to ship language that is strictly incorrect in our CP/CPS so I think we should fix it and not mimic the BRs here.

Reply:
Same suggestion from #245. Perhaps we can lift the phrase "every requested identifier" from Section 4.2.2.

Reply:
I like using "requested". I've updated this PR, let me know what you think.